handlers.go

```diff
@@ -4147,6 +4147,7 @@ type collectionListQueryRes struct {
 	Cid    util.DbCID
 	Size   int64
 	Path   *string
+	Dir    bool
```

Comment:
You'll also want to wire this into `handleCreateContent` and use it on the shuttle, `cmd/estuary-shuttle/main.go` line ~1184

Reply:
Donezo!